modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -235,6 +235,12 @@ public ObjectDefinition postObjectDefinition(
 			throw new ObjectDefinitionEnableLocalizationException();
 		}
 
+		if (Validator.isNotNull(objectDefinition.getEnableObjectEntryDraft()) &&
+			!FeatureFlagManagerUtil.isEnabled("LPS-181663")) {
```

Comment:
The feature flag must be the LPS from the epic. In your case, I believe it is: https://liferay.atlassian.net/browse/LPS-187846.

Reply:
:-1: